examples/kingworld/gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+node_modules
```

Comment:
Could you use the same gitignore as the ``next`` template? It's just to have some consistency between the templates.